.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```
@@ -0,0 +1,59 @@
+name: 🚀 Feature request
+description: Suggest an idea for improving Wagtail
```

Discussed at core meeting. What about something like
```suggestion
description: Suggest an idea for improving Wagtail, or its developer experience
```